mdsuite/database/simulation_database.py

```diff
@@ -424,7 +424,7 @@ def change_key_names(self, mapping: dict):
                 if item in mapping:
                     db.move(item, mapping[item])
 
-    def load_data(self, path_list: list = None, select_slice: np.s_ = None, dictionary: bool = False,
+    def load_data(self, path_list: list = None, select_slice: np.s_ = np.s_[:], dictionary: bool = False,
```

Comment:
We need to keep this in mind during our testing phase as there could be some unwanted results.